### ISSUE TITLE
Nikon D90 Firmware

### ISSUE THREAD

**Issue by GregLomason** [simeonpilgrim/nikon-firmware-tools#143]
The only issue is that there isn't a firmware update/patch for the Nikon D90. Could this be remedied? @simeonpilgrim

**Comment by simeonpilgrim** [comment]
> The only issue is that there isn't a firmware update/patch for the Nikon D90

Well the file on the Nikon site: https://downloadcenter.nikonimglib.com/en/download/fw/9.html is a valid A firmware. So that part is "something that could be worked with".

The second part, "there are no patches" this is true. I have no desire/need to make patches for a camera I do not have. But it could be done by others, within the limits of what the camera can do, though.

**Comment by GregLomason** [comment]
This is true. I downloaded the one from the Nikon website. I would like to enhance/increase the video capabilities of the Nikon D90 to say Full HD at 12 FPS or something similar to what your code allows others to do with those listed on the website application. Alas, I'm not well versed enough or familiar enough with the code to use it in a way to do so for myself. A YouTube tutorial would be amazing, though I understand if that is by no means part of your plans. I realize the benefit and thoughtful consideration you've placed into creating this I'm just a little sore I won't be able to use it too lol

**Comment by simeonpilgrim** [comment]
nod, understood, perhaps my privilege is leaking, but 12 years ago, when I purchased my D5100 and wished for the unbridled power of the D90 or D7000 (ignoring the pro models), I too wanted more from my camera. 

Then I got a D7000 and it was nicer, then I got a D750, and it does what I want, and what it does not do, it cannot do. And recently I picked up a old D5100 for 100NZD (60USD) and that does better video than the D90 offers, so I just don't see why you wouldn't just pick-up something that does more of what you want, verse spending hundereds of hours to make something get 10% at something it just doesn't do that good.

I mean sure, if you want minor changes to allow this, or default to that. That might be viable. But better video, is just not in this camera's nature.

**Comment by GregLomason** [comment]
Thank you for your thoughtful and well-considered reply. I understand where you're coming from and can gratefully say thank you for that perspective. Sometimes I get caught up in what I can't have I forget to look outside the box a little bit. My NikonD90 was left to me by my brother-in-law when he passed, I thought to put as much of "my own" spins on it as I could to both keep it true to how he used it while enhancing it to be more complimentary to my needs. New cameras are on the wish list, so maybe once I'm ready and a little more prepared to put the Nikon on the shelf I'll look for those and return to your web app for some "hacks." I appreciate your time and you're response, have a wonderful day, and if we don't speak again soon a great year!

**Comment by toups** [comment]
If looking for a "new" camera, don't forget the used market.  Recently went from a D90 to a full frame D610 which was a nice upgrade at the fraction of the cost of the latest models.

**Comment by simeonpilgrim** [comment]
exactly, this was my point about the D5100, it video is way better than the D90, and crazy cheap.
